ruby-gem/Rakefile

```diff
@@ -36,14 +35,31 @@ def build
 end
 
 task :build do
```

Comment:
The task :build conflicts with the Bundler tasks:

```
$ rake build # should build the gem
```

I recommend changing to `:build_server`